WhisperAPI/WhisperAPI/Models/MessageSuggestion.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace WhisperAPI.Models
+{
+    /// <summary>
+    ///  Wrapper arround message string because entity framework doesn't allow primitive type in collection
```

Comment:
You are right, this sucks, we will see next sprint what we do about that (perhaps not using Entity Framework).